tests/models/volunteers.test.js

```diff
@@ -0,0 +1,72 @@
+const chai = require('chai');
+const expect = chai.expect;
+var Department = require('../../models/volunteers').Department;
+var Volunteer = require('../../models/volunteers').Volunteer;
+var User = require('../../models/user').User;
+
+describe('Volunteer module db model', () => {
```

Comment:
Let's talk when we meet.These tests are not for the API but for the model relations mapping. Most probably most of the mappings are not needed anymore.